blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -141,8 +141,12 @@ def search_func(self, model: Gtk.TreeModel, column: int, key: str, tree_iter: Gt
         return True
 
     def filter_func(self, _model: Gtk.TreeModel, tree_iter: Gtk.TreeIter, _data: Any) -> bool:
-        no_name = self.get(tree_iter, "no_name")["no_name"]
-        if no_name and self.Config["hide-unnamed"]:
+        row = self.get(tree_iter, "no_name", "device")
+        no_name = row["no_name"]
+        device = row["device"]
+        klass = get_minor_class(device["Class"]) if device is not None else None
+
+        if no_name and self.Config["hide-unnamed"] and klass not in (_("Keyboard"), _("Pointing"), _("Combo")):
```

Comment:
That's a weird check. It's basically just the Peripheral major class. I'm not sure, though, if we should actually include pointing devices or just target keyboards (and possibly combos) specifically.

Reply:
dropped pointing from the list